src/components/organisms/StampGenerator.vue

```diff
@@ -37,17 +37,26 @@ export default defineComponent({
       if (this.text === '') {
         return
       }
-      // // 横幅を取得
-      // const text_width = this.ctx.measureText(this.text).width
-      // 横幅に合わせて、横方向の倍率を調整
-      // ctx.scale(1, 1)
+
       // 描画
       const [red, green, blue] = this.colors
       ctx.strokeStyle = `rgb(${red}, ${green}, ${blue})`
-      ctx.font = 'bold 48px ' + this.picked
+      ctx.font = `bold ${w}px ${this.picked}`
+      ctx.textBaseline = 'top'
+
       const lines = this.text.replace(/\r/g, '').split('\n')
+
       for (let i = 0; i < lines.length; i++) {
-        ctx.fillText(lines[i], 0, 48 * (i + 1))
+        // 横幅から
+        const text_width = ctx.measureText(lines[i]).width
+        if (text_width === 0) {
```

Comment:
```suggestion
        const textWidth = ctx.measureText(lines[i]).width
        if (textWidth === 0) {
```
他と合わせてcamelCaseにしてもらえると:pray: